src/styles/_mixins.less

```diff
@@ -0,0 +1,58 @@
+/* HelixUI Mixins */
+
+/* ----- GRID ----- */
+.make-grid-columns (@prefix, @n) when (@n > 0 ) and (@prefix = xs) {
+  .span-@{n},
+  .span-@{prefix}-@{n} {
+    @width: (@n * @grid-column-unit);
+    min-width: @width;
+    width: @width;
+  }
+  .make-grid-columns(@prefix, (@n - 1));
+}
+.make-grid-columns (@prefix, @n) when (@n > 0 ) and not (@prefix = xs) {
+  .span-@{prefix}-@{n} {
+    @width: (@n * @grid-column-unit);
+    min-width: @width;
+    width: @width;
+  }
+  .make-grid-columns(@prefix, (@n - 1));
+}
+.make-grid-offsets (@prefix, @n) when (@n > 0) and (@prefix = xs) {
+  .offset-@{n},
+  .offset-@{prefix}-@{n} {
+    margin-left: (@n * @grid-column-unit) !important;
+  }
+  .make-grid-offsets(@prefix, (@n - 1));
+}
+.make-grid-offsets (@prefix, @n) when (@n > 0) and not (@prefix = xs) {
+  .offset-@{prefix}-@{n} {
+    margin-left: (@n * @grid-column-unit) !important;
+  }
+  .make-grid-offsets(@prefix, (@n - 1));
+}
+.make-grid-orders (@prefix, @n) when (@n > 0) and (@prefix = xs) {
+  .order-@{n},
+  .order-@{prefix}-@{n} {
+    order: @n;
+  }
+  .make-grid-orders(@prefix, (@n - 1));
+}
+.make-grid-orders (@prefix, @n) when (@n > 0) and not (@prefix = xs) {
+  .order-@{prefix}-@{n} {
+    order: @n;
+  }
+  .make-grid-orders(@prefix, (@n - 1));
+}
+.make-grid (@prefix) {
+  .make-grid-columns(@prefix, @grid-column-count);
+  .make-grid-offsets(@prefix, @grid-column-count);
+  .make-grid-orders(@prefix, @grid-column-count);
+}
+
+/* ----- Custom Props ----- */
+// EXAMPLE: .customProp(color, --my-var, #333);
+.customProp (@prop, @val, @fallback) {
```

Comment:
Will be necessary for applying custom properties in a backward-compatible manner.